src/remotestorage.js

```diff
@@ -270,7 +270,7 @@
   };
 
   RemoteStorage.config = {
-    logging: false,
+    logging: true,//mrhTODO disable logging
```

Comment:
This shouldn't be committed and ideally be done from the app/client code instead of here.